lib/livebook/session.ex

```diff
@@ -2549,6 +2565,21 @@ defmodule Livebook.Session do
   defp handle_action(state, _action), do: state
 
   defp start_evaluation(state, cell, section, evaluation_opts) do
+    evaluation_users =
+      case state.data.mode do
+        :default -> Map.values(state.data.users_map)
+        :app -> if(state.deployed_by, do: [state.deployed_by], else: [])
+      end
+
+    Logger.info(
+      """
+      Evaluating code
+        Session mode: #{state.data.mode}
+        Code: #{inspect(cell.source, printable_limit: :infinity)}\
+      """,
```

Comment:
Good candidate for IO data, since it can be large?

```suggestion
      ["""
      Evaluating code
        Session mode: #{state.data.mode}
        Code: \
      """,
      inspect(cell.source, printable_limit: :infinity)
      ]
```